otter/log/formatters.py

```diff
@@ -261,6 +271,9 @@ def error_formatting_observer(event):
                 message = repr(excp)
                 event['traceback'] = event['failure'].getTraceback()
                 event['exception_type'] = excp.__class__.__name__
+                details = serialize_to_jsonable(excp)
+                if details != repr(excp):
```

Comment:
I see now. I only suggest changing the "if details != repr(excp):" to "if details != message" then. Sorry & thanks!